docs/Background.md

```diff
@@ -1,6 +1,12 @@
 # Background
 
-The Form Builder builds a JSON schema for use with the [JSON Schema Form component](https://react-jsonschema-form.readthedocs.io/en/latest/). It abstracts away the manual coding element for a flexible visual form building experience. 
+The Form Builder builds a JSON schema for use with the [JSON Schema Form component](https://react-jsonschema-form.readthedocs.io/en/latest/). It abstracts away the manual coding element for a flexible visual form building experience.
+
+One of the key abstractions is rendering form elements by parsing the `properties`, `definitions`, and `dependencies` sections of standard JSON Schema form code into discrete elements, that can be rerranged and edited in a manner similar to other standard form building solutions.
```

Comment:
Typo, should be `rearranged`. (While you're at it, you could remove the `,` prior to `that can be rerranged [sic]`.